altair/utils/schemapi.py

```diff
@@ -327,11 +358,9 @@ def to_dict(self, validate=True, ignore=None, context=None):
 
         Parameters
         ----------
-        validate : boolean or string
+        validate : boolean
             If True (default), then validate the output dictionary
```

Comment:
I find it confusing that there are two parameters names `validate`, one in the private method `_to_dict` which we manually set to `False` everywhere in this PR, and one in `to_dict` which is actually controlled by the user. Could we remove the `validate` param of the  `_todict` private function and just pass `False` on line 129 where `to_dict` is called inside `_todict`? It seems like this will never be `True` with the changes in this PR.

Reply:
Good point, done.